### PR TITLE
Project Setup

### PR DESCRIPTION
**Pull request by Dandush03**
# Finished 3rd JS Project
### Used:
- JavaScript
- ES6
- HTML 5
- SASS
- ESLint

### CODE REVIEW

**Review by jofer86 — CHANGES_REQUESTED** [review]
## :wave: :wave: Hello!, I'm Jorge, and  I'm here to review this Pull Request, let's take a look shall we :eye: :eye: :eye: 
- Changes marked as **[OPTIONAL]** are not hard requirements but are nice to have.
- Firstly we need to modularize our code a little bit more, even though you are doing the correct thing appending the components to your main div, the entire app should be constructed with modules, and every module needs to export the component. IE using the export import syntax. [Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/tab_requirement.png)
- The module exports need to export, aside from the content, the styles, now you could take two approaches here, do the styling individually for each module, and have those files live in a separate folder and have them come together, or you could do your styling inside your JS files.

# `Project Status: Almost There!` :goal_net: :goal_net: :goal_net: 

--------------------------------------------------------------------
Cheers, 
Jorge Fernando Rincón Figueroa.
Technical Support Engineer.
**Slack: @jofer86**

**Comment by Dandush03** [comment]
# Hey! Don't know  how i miss that hehe.

It is done, I modulated my code as much as I could.

Hope I did a good job.

##### Looking forward for your next review

**Comment by Dandush03** [comment]
# Hey!
## Don't know  how i miss that hehe. :wink: 

It is done, I modulated my code as much as I could.

Hope I did a good job.

##### Looking forward for your next review

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+        "Atomics": "readonly",
+        "SharedArrayBuffer": "readonly"
+    },
+    "parserOptions": {
+        "ecmaVersion": 2018,
+        "sourceType": "module"
+    },
+    "rules": {
+    }
+}
```
Comment (shloch) [inline .eslintrc.json]: Please use the linting config file provided by MICROVERSE --> [ [see config](https://github.com/microverseinc/linters-config/blob/master/javascript/.eslintrc.json) ]

**Review by shloch — CHANGES_REQUESTED** [review]
## hello @Dandush03 

We're good to go, great job............

### [Required Changes]
Let's just use the correct linting file and we're good to go
### Happy coding

**Comment by Dandush03** [comment]
# HEY!!! :wave: 

## Thanks for you review.

#### I hope you understand that you asking me to disable some rule wich actually will not change ANYTHING on my project. :weary: 

# Have a good day. 

# Happy Coding :computer:

**Review by azdrenymeri — APPROVED** [review]
### Hello @Dandush03  :wave: 
#### The linter `config` is done correctly. :heavy_check_mark: 
#### You can merge it :+1:

**Pull request merged**